docker-compose.yml

```diff
@@ -56,7 +56,7 @@ services:
         - type=registry,ref=us-east1-docker.pkg.dev/firezone-staging/cache/web:main
       args:
         APPLICATION_NAME: web
-    image: us-east1-docker.pkg.dev/firezone-staging/firezone/web:${VERSION:-main}
+    image: ${WEB_IMAGE:-us-east1-docker.pkg.dev/firezone-staging/firezone/web}:${WEB_TAG:-main}
```

Comment:
👍  The web, client, and gateway images were split up so that we can do the compat tests, previously everything was just VERSION, right? Looks good.